projects/authentication/docs/introduction.md

```diff
@@ -0,0 +1,9 @@
+# Introduction
+
+This space gives an overview of the possible authentication methods on publiq's APIs.
+
+In the past, almost every one of our APIs used a different authentication method. To provide a consistent experience and reduce the work needed to integrate with multiple of our APIs, we have evolved our APIs to support a shared authentication approach.
+
+To start, you will need a set of client credentials consisting of a **client id** and **client secret** which we will provide you. You can then use these in 3 ways to authenticate your requests to our APIs, depending on the API(s) you integrate with and your function use case. See the [authentication methods overview](./methods.md) for more info about the various ways that you can authenticate your requests.
```

Comment:
... and your **function** use case. ...

I have the feeling that the word "function" is redundant. Could it be left away?

Reply:
Was meant to be _functional_ use case 🙈  But maybe it's also fine without it yes!